### PR TITLE
[Show][BGP] Show BGP Change for no neighbor scenario

### PR DESCRIPTION
**Pull request by developfast**
#### What I did
Changed the show bgp summary for both ipv4 and ipv6 to show basic bgp switch information. This was done in order for hardware proxy to set the correct settings for bgp connections when no neighbors are present, and prevent errors. This was a blocking issue which prevents image upgrades for backend storage ToRs which have no bgp neighbors. 

#### How I did it
Checked for when a bgp neighbor is present using vtysh, if not, display basic bgp information. 

#### How to verify it
Copied the respective bgp util files onto a switch and ran `show ip bgp summary` and `show ipv6 bgp summary` when there are no bgp neighbors and see that it displays the correct information.

#### Previous command output (if the output of a command-line utility has changed)
```
~$ show ip bgp summary
No IPv4 neighbor is configured

~$ show ipv6 bgp summary
No IPv6 neighbor is configured
```

#### New command output (if the output of a command-line utility has changed)
```
~$ show ip bgp summary

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 12811
RIB entries 0, using 0 bytes of memory
Peers 0, using 0 KiB of memory
Peer groups 0, using 0 bytes of memory


Neighbor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down    State/PfxRcd    NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------

Total number of neighbors 0

~$ show ipv6 bgp summary

IPv6 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 8972
RIB entries 0, using 0 bytes of memory
Peers 0, using 0 KiB of memory
Peer groups 0, using 0 bytes of memory


Neighbor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down    State/PfxRcd    NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------

Total number of neighbors 0
```

### CODE REVIEW

File: utilities_common/bgp_util.py [hunk 1]
```diff
@@ -227,21 +229,30 @@ def get_bgp_summary_from_all_bgp_instances(af, namespace, display):
...
         except ValueError:
             ctx.fail("bgp summary from bgp container not in json format")
 
-        # exit cli command without printing the error message
-        if key not in cmd_output_json:
-            click.echo("No IP{} neighbor is configured".format(af))
-            exit()
+        # no bgp neighbors found so print basic device bgp info
+        if key not in cmd_output_json or not has_bgp_neighbors:
+            vtysh_cmd = "show ip bgp json"
```
Comment (arlakshm) [inline utilities_common/bgp_util.py:246]: set the has_bgp_neighbors when the cmd_output_json is not as expected?

File: tests/bgp_commands_test.py [hunk 2]
```diff
@@ -98,11 +98,35 @@
...
+
+Neighbhor    V    AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ    Up/Down    State/PfxRcd    NeighborName
+-----------  ---  ----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
+
+Total number of neighbors 0
 """
 
 show_error_no_v4_neighbor = """\
-No IPv4 neighbor is configured
+
```
Comment (arlakshm) [inline tests/bgp_commands_test.py:117]: Please add tests for multi asic as well

**Review by arlakshm — COMMENTED** [review]
as comments

File: utilities_common/bgp_util.py (hunk 1 above)
Comment (developfast) [reply utilities_common/bgp_util.py:245]: done

File: tests/bgp_commands_test.py (hunk 2 above)
Comment (developfast) [reply tests/bgp_commands_test.py:117]: added

**Review by arlakshm — APPROVED** [review]

**Comment by yxieca** [comment]
This change has been back ported to 202012 branch. Needs to keep it consistently in all future branches.

**Comment by StormLiangMS** [comment]
@developfast  ADO number?

**Comment by developfast** [comment]
@StormLiangMS ADO # 24341951

**Pull request merged**